src/runner.rs

```diff
@@ -0,0 +1,63 @@
+use crate::constants::EXIT_FAILURE;
+use crate::scanner::{BundleRepo, FileSet, FileSetCounter};
+use crate::types::{RunResult, Test};
+use junit_parser;
+use std::fs::metadata;
+use std::process::Command;
+use std::process::Stdio;
+use std::time::SystemTime;
+
+pub async fn run_test_command(
+    repo: &BundleRepo,
+    command: &String,
+    args: Vec<&String>,
+    output_paths: Vec<&String>,
+) -> anyhow::Result<RunResult> {
```

Comment:
If this function fails to process output of the command, what will happen? Do we ever want to override exit code of the child command?

Reply:
We should only override the exit code if the only failures parsed were quarantined.